test/src/main/java/org/apache/accumulo/test/mapreduce/AccumuloOutputFormatIT.java

```diff
@@ -164,11 +162,9 @@ public void testMR() throws Exception {
       assertNull(e1);
 
       try (Scanner scanner = c.createScanner(table2, new Authorizations())) {
-        Iterator<Entry<Key,Value>> iter = scanner.iterator();
-        assertTrue(iter.hasNext());
-        Entry<Key,Value> entry = iter.next();
-        assertEquals(Integer.parseInt(new String(entry.getValue().get())), 100);
-        assertFalse(iter.hasNext());
+        int actual = scanner.stream().map(Entry::getValue).map(Value::get).map(String::new)
+            .map(Integer::parseInt).collect(onlyElement());
```

Comment:
It's slightly shorter if you just call parseInt on the only element (some suggestion applies elsewhere in this PR):

```suggestion
        int actual = Integer.parseInt(scanner.stream().map(Entry::getValue).map(Value::get).map(String::new).collect(onlyElement()));
```

Reply:
Personally, it seems to be a bit more intuitive or readable, when using multiple `.map()` operations, to just keep with that pattern and do the `parseInt` within a `.map()` as well, even if it is slightly longer.

Reply:
This reminds me of my comment in the description of this PR:
> I also wanted to see if anyone had an opinion on which of the following equivalent options is better:
> 
>     1. `scanner.stream().map(Entry::getValue).map(Value::get).map(String::new).map(Integer::parseInt).collect(onlyElement());`
> 
>     2. `scanner.stream().map(entry -> Integer.parseInt(new String(entry.getValue().get()))).collect(onlyElement());`

I'm starting to think I prefer option 1 over option 2. It seems easier to read and keep track of what is happening especially with all the nested parentheses in option 2.

Reply:
I've used both. I prefer option 1, if all things are equal, but use option 2 on a case-by-case basis if it's shorter, more clear, or have reason to think it might perform better.